Resources/ChartData/rfs6-errors.json

```diff
@@ -2,54 +2,93 @@
   {
     "id" : "all",
     "name" : "All packages",
+    "total" : 3396,
     "values" : [
       {
         "date" : "2024-05-04",
+        "toolchainId" : "org.swift.600202404221a",
+        "toolchainLabel" : "Swift 6.0 Development Snapshot 2024-04-22 (a)",
```

Comment:
I'm a little confused that `total` and the toolchain labels are new to these files. Didn't we add those newer versions to the server project yet?

Reply:
I [added the totals manually](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/pull/3144/files#diff-818a93b4aedf73eceafda54d54f87bd518775dcd87ebf3deaa1c2222a842d544) for the top chart earlier but didn't bother with them for the errors chart as they're ignored.

When I added them to the tool, I realised they also got generated for that chart too, and it's no problem, they just don't get used there.

Reply:
I don't think I knew about `toolchainId` and `toolchainLabel ` 🫣 I remember discussing them but must have missed them going in. I'll bring them through now.